SCons/Tool/ninja/ninja.xml

```diff
@@ -105,45 +105,54 @@ See its __doc__ string for a discussion of the format.
             <item>SHLINKCOM</item>
             <item>PROGSUFFIX</item>
             <item>PRINT_CMD_LINE_FUNC</item>
-            <!--            <item>TEMPFILE</item>-->
+            <!--<item>TEMPFILE</item>-->
 
-            <!-- TODO: Document these
-            <item>NINJA_MAX_JOBS</item>
-            <item>NINJA_SKIP</item>
-            <item>NINJA_CUSTOM_HANDLERS</item>
--->
+            <!-- TODO: Document these -->
+            <!-- <item>NINJA_MAX_JOBS</item> -->
+            <!-- <item>NINJA_SKIP</item> -->
+            <!-- <item>NINJA_CUSTOM_HANDLERS</item> -->
 
         </uses>
     </tool>
 
     <builder name="Ninja">
         <summary>
             <para>
-                &b-Ninja; is a special builder which
-                adds a target to create a ninja build file.
+                A special builder which
+                adds a target to create a Ninja build file.
                 The builder does not require any source files to be specified.
             </para>
             <note>
                 <para>This is an experimental feature. To enable it you must use one of the following methods
                 </para>
 
-<!--                NOTE DO NOT INDENT THE FOLLOWING AS IT WILL ALTER THE FORMATTING-->
+<!--            NOTE DO NOT INDENT example_commands CONTENTS AS IT WILL ALTER THE FORMATTING-->
                 <example_commands>
-    # On the command line
-    --experimental=ninja
+# On the command line
+--experimental=ninja
 
-    # Or in your SConstruct
-    SetOption('experimental', 'ninja')
+# Or in your SConstruct
+SetOption('experimental', 'ninja')
                 </example_commands>
 
                 <para>This functionality is subject to change and/or removal without deprecation cycle.</para>
 
-
                 <para>
-                    To use this tool you must install pypi's <ulink url="https://pypi.org/project/ninja/">ninja
-                    package</ulink>.
-                    This can be done via
-                    <command>pip install ninja</command>
+                    To use this tool you need to install the &Python; &ninja; package,
+                    as the tool by default depends on being able to do an
+                    <systemitem>import</systemitem> of the package
+                    (although see &cv-link-__NINJA_NO;).
+                    This can be done via:
```

Comment:
I think we should be fine with the previous pip install, rather than listing out for each platform/packaging system.. At least for the tool which ends up in man.

Reply:
Previous? Or you mean the userguide one?

Reply:
I trimmed this down to one example.